Simvex_FE/src/pages/Study-bp.jsx

```
@@ -0,0 +1,455 @@
+import arrow from "../assets/arrow.svg";
+import X from "../assets/X.svg";
+import tr from "../assets/tr.svg";
+import plus from "../assets/Plus.png";
+import minus from "../assets/Minus.svg";
+import star from "../assets/star.svg";
+import glass from "../assets/glass.svg";
+import oplus from "../assets/oplus.svg";
+
+const PartModel = ({ url, position, rotation, scale = 1, explosionFactor }) => {
+  try {
+    const { scene } = useGLTF(url);
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)